easybuild/easyconfigs/c/CAT-BAT/CAT-BAT-5.2.3-GCC-10.3.0.eb

```diff
@@ -0,0 +1,48 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+easyblock = 'Tarball'
+
+name = 'CAT-BAT'
+version = '5.2.3'
+
+homepage = 'https://github.com/dutilh/CAT'
+description = """Tool for taxonomic classification of contigs and metagenome-assembled genomes (MAGs)."""
+# software_license = 'LicenseMIT'
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/dutilh/CAT/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['19a8eb28c0ebd3fc5a04b006d305a916dd1af00a32b70d2f30df35f32ca2929b']
+
+builddependencies = [('binutils', '2.36.1')]
```

Comment:
`binutils` is not required when using `GCC` as toolchain (only required when using `GCCcore` toolchain)